src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -140,7 +140,7 @@ def _run_sql(self, query: str) -> BigQueryRows:
             except Forbidden as e:
                 if 'Exceeded rate limits' in e.message and delay is not None:
                     log.warning('Exceeded BigQuery rate limit during attempt %i/%i. Retrying in %is.',
-                                attempt + 1, len(delays), delay, exc_info=e)
+                                attempt + 1, len(delays) + 1, delay, exc_info=e)
```

Comment:
@hannes-ucsc I changed the warnings to ensure the number of retries in the log looks correct.

do something; attempt 1/5; wait 10s 
do something; attempt 2/5; wait 20s 
do something; attempt 3/5; wait 40s 
do something; attempt 4/5; wait 80s 
do something; exception

rather than `attempt 4/4` and there is no mention of the 5th retry.

Reply:
Looks good.